### PR TITLE
Http to https automatically redirect

### PR DESCRIPTION
**Pull request by Menkveld-24**
Docker nginx is modified to automatically redirect non-https requests to https so you don't have to type https://... every time

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -39,9 +39,12 @@ services:
     ports:
       - 127.0.0.1:${DOCKER_HTTP_PORT}:443
     volumes:
-      - ./docker/nginx/nginx.conf:/etc/nginx/nginx.conf
+      - ./docker/nginx/nginx.conf:/etc/nginx/templates/nginx.conf.template
```
Comment (Jessseee) [inline docker-compose.yml:42]: Why are you changing this directory?

Comment (Menkveld-24) [reply docker-compose.yml:42]: I didn't want to hardcode port 8080 for the redirect and I can't use the incoming port (it thinks it comes from 443). So I wanted to use the same port variable as in the .env. But nginx is unable to resolve the set environment variable. Here comes envsubst, it is built into nginx (see https://hub.docker.com/_/nginx at "Using environment variables in nginx configuration"), so nginx substitutes the variable and then replaces the file. But in order to do that the file first needs to be in the dedicated directory for this

Comment (Jessseee) [reply docker-compose.yml:42]: ohboi, well if it works...

**Review by Jessseee — APPROVED** [review]
LGTM

**Pull request merged**